mlir-tensorrt/build_tools/cmake/Dependencies.cmake

```diff
@@ -100,6 +104,28 @@ function(download_tensorrt)
 
   set(TRT_VERSION "${ARG_VERSION}")
 
+  # Handle TensorRT 8 versions. These are publicly accessible download links.
```

Comment:
This is for TRT9

Reply:
I was wrong, thanks for figuring this out

Reply:
This also works for TRT 8 versions with slight modifications.
Right now I am allowing only TRT 8.6.1.6